driver_cpl/cime_config/config_component.xml

```diff
@@ -643,7 +643,7 @@
     <file>env_build.xml</file>
     <desc>Operating system - DO NOT EDIT UNLESS for userdefined machine - ignored once Macros has been created.</desc>
   </entry>
-
+<!--
```

Comment:
I don't think that we need COMPILERS and MPILIBS in the case.  I've commented them out, but can remove them if you prefer.